cms/grading/Sandbox.py

```diff
@@ -186,6 +187,8 @@ class SandboxBase(object):
 
     """
 
+    __metaclass__ = ABCMeta
```

Comment:
This looks like neither of the two recommended ways of doing it (reading https://docs.python.org/3/library/abc.html) - is there a reason not to follow documentation?

Reply:
There are good reasons not to follow the Py3 doc and follow the Py2 instead: https://docs.python.org/2.7/library/abc.html

:P